services/apps/portal/src/application/application-parcel/application-parcel.entity.ts

```diff
@@ -121,9 +121,6 @@ export class ApplicationParcel extends Base {
   @OneToMany(
     () => ApplicationParcelDocument,
     (appParcelDocument) => appParcelDocument.applicationParcel,
-    {
```

Comment:
Why do we need to keep a document that is not linked to anything on the portal?

Reply:
Woops! I think I added this to Owners above but ended up removing it and accidentally removed this one also. Good catch